js/data.js

```diff
@@ -6,3 +6,45 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
+var formInput = document.querySelector('.input-form');
+
+formInput.addEventListener('submit', submitUserInput);
+
+function submitUserInput(event) {
+  event.preventDefault();
+  var image = document.querySelector('img');
+  var inputForm = document.querySelector('.input-form');
+  var input = inputForm.elements;
+  var dataObject = {};
+  for (var i = 0; i < input.length; i++) {
+    if (input[i].name === 'title') {
+      dataObject.title = input[i].value;
+    } else if (input[i].name === 'photo-url') {
+      dataObject.url = input[i].value;
+    } else if (input[i].name === 'notes') {
+      dataObject.notes = input[i].value;
+    }
+  }
+  data.nextEntryId += 1;
+  dataObject.nextEntryId = data.nextEntryId;
```

Comment:
These two steps are backwards, you should use the value before you increment it. Also, `nextEntryId` is not an accurate name for your property. It makes sense in the data model since it is saving the next id, but here it is just an id. `entryId` would be a more appropriate name for your property